gh-126220: Fix crash on calls to `_lsprof.Profiler` methods with 0 args (backportable) (#126271)
python/cpython: Misc/NEWS.d/next/Library/2024-10-31-14-06-28.gh-issue-126220.uJAJCU.rst

> Fix crash of ``cProfile.Profile`` and ``_lsprof.Profiler`` when their

Reviewer: ```suggestion
Fix crash in :class:`cProfile.Profile` and :class:`!_lsprof.Profiler` when their
```
